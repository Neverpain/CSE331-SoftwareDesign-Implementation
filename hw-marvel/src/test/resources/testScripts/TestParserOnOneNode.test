# Tests that the makeGraph method properly creates a one node
# graph with no edges or anything, prints it out

LoadGraph g oneNodeGraph.csv

ListNodes g
ListChildren g n1

