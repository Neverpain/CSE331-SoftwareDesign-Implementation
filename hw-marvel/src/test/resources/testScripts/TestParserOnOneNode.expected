# Tests that the makeGraph method properly creates a one node
# graph with no edges or anything, prints it out

loaded graph g

g contains: n1
the children of n1 in g are: